packages/paste-website/src/pages/layout/media-object/index.mdx

```diff
@@ -0,0 +1,287 @@
+  {`<MediaObject as="div" verticalAlign="center">
+  <MediaFigure
+    as="div"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+  <MediaBody as="div">
+    Some Text
+  </MediaBody>
+</MediaObject>`}
+</LivePreview>
+
+### MediaFigure at the end
+
+To place the `MediaFigure` after the `MediaBody`, at the end of the `MediaObject`, place the `MediaFigure` after the `MediaBody` in the DOM. Be sure to set the `align` prop on the `MediaFigure` to "end" to reverse the spacing.
+
+<LivePreview scope={{Box, MediaObject, MediaFigure, MediaBody}} language="jsx">
+  {`<MediaObject as="div" verticalAlign="center">
+  <MediaBody as="div">
+    Some Text
+  </MediaBody>
+  <MediaFigure
+    as="div"
+    align="end"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+</MediaObject>`}
+</LivePreview>
+
+### Double figure
+
+To have a `MediaFigure` at either end of the `MediaObject` add a `MediaFigure` before and after the `MediaBody`, setting the second `MediaFigure` to `align` "end".
+
+<LivePreview scope={{Box, MediaObject, MediaFigure, MediaBody}} language="jsx">
+  {`<MediaObject as="div" verticalAlign="center">
+  <MediaFigure
+    as="div"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+  <MediaBody as="div">
+    Some Text
+  </MediaBody>
+  <MediaFigure
+    as="div"
+    align="end"
+    spacing="space40"
+  >
+    <Box backgroundColor="colorBackgroundDark" borderRadius="borderRadius20" height="sizeIcon100" width="sizeIcon100"></Box>
+  </MediaFigure>
+</MediaObject>`}
+</LivePreview>
+
+## Paste Examples
+
+Places where you are likely to see this pattern used in Paste include:
+
+### The Alert component
+
+The Alert is a classic Media Object layout. The Status level icon is the Media Figure, and the alert description is the Media Body.
+
+<LivePreview scope={{Alert}} language="jsx">
+  {`<Alert variant="warning">
+  <strong>Caution:</strong> You may never unsee this pattern across the web!
+</Alert>`}
+</LivePreview>
+
+### Submenu component
+
+A Submenu Item uses a Media Object where the Media Figure is aligned to the end. It is used to show the user there are further options to a particular menu item with the use of a chevron icon.
+
+<LivePreview
+  scope={{
+    useMenuState,
+    Menu,
+    MenuButton,
+    SubMenuButton,
+    MenuItem,
+    MenuSeparator,
+    ChevronDownIcon,
+  }}
+  noInline
+>
+  {subMenuExample}
+</LivePreview>
+
+---
+
+## Usage Guide
+
+In the Paste Media Object the size of the figure is determined by the content that is placed inside of it. The body will fill the remaining space in the row.
```

Comment:
```suggestion
The size of `MediaFigure` is determined by the content that is placed inside of it. `MediaBody` will fill the remaining space in the row.
```

Reply:
Actually..I don't even know if this is needed here. It's basically covered in Basic Media Object, except for the part about how MediaFigure is sized.